README.md

```diff
@@ -2,19 +2,41 @@
 
 1.
   * Create a Human class that takes in a name and age.
-  * Add the function `ageOneYear` that ages the human. 
+  * Add the function `ageOneYear` that ages the human.
   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
   * Create an instance of the Human class.
     * console log your humans age
     * call ageOneYear on your human
     * console log their age again.
     * call eating on your human.
-    
-2. 
+
+    class Human{
+      constructor(name, age){
+        this.name = name;
+        this.age = age;
+      }
+      ageOneYear(num){
+        this.age++
+        return this.age
+      }
+      eating(str){
+        return `mmm, mmm, mmm, I'm love'n it`
+      }
+      }
+    let zyon = new Human("Zyon", 3)
+
+    console.log(zyon.age)
+    console.log(zyon.ageOneYear())
+    console.log(zyon.eating())
+
```

Comment:
Small note: variable names that are classes should begin with an uppercase.
Great job with this assignment!